articles/code_coverage.dd

```diff
@@ -35,12 +35,12 @@ $(LI It can be used to track down why a particular section of code
 $(LI Since execution counts are given for each line, it is possible
      to use the coverage analysis to reorder the basic blocks in
      a function to minimize jmps in the most used path, thus
-     optimizing it.)
-)
+     optimizing it. This is known as $(LINK2 https://en.wikipedia.org/wiki/Profile-guided_optimization, profile-guided optimization)
+))
 
 $(P Experience with code coverage analyzers show that they dramatically
 reduce the number of bugs in shipping code.
-But it isn't a panacea, a code coverage analyzer won't help with:)
+But it isn't a panacea, a code coverage analyzer won't help (Unlike Valgrind or the sanitizers available in GCC and LLVM) with:)
```

Comment:
Looks like I missed this one earlier: "Unlike" should be "unlike".

Reply:
done